### ISSUE TITLE
hrvst time-entries / Content-Type header required to be application/json, got nil

### ISSUE THREAD

**Issue by bjohas** [kgajera/hrvst-cli#3]
The operational `hrvst time-entries delete` doesn't seem to work:
```
> hrvst time-entries delete -t 1234
Content-Type header required to be application/json, got nil
```

**Comment by kgajera** [comment]
Thanks for reporting this as well as the other feature requests! I will release a new version tomorrow night with all these changes.

**Comment by kgajera** [comment]
This has been fixed in [v1.4.0](https://github.com/kgajera/hrvst-cli/releases/tag/v1.4.0).

**Comment by bjohas** [comment]
@kgajera That's amazing - thank you so much!
